Realm/RLMRealmConfiguration.h

```diff
@@ -60,6 +60,9 @@ RLM_ASSUME_NONNULL_BEGIN
 /// The block which migrates the Realm to the current version.
 @property (nonatomic, copy, nullable) RLMMigrationBlock migrationBlock;
 
+/// A custom schema to use for the Realm.
+@property (nonatomic, copy, nullable) RLMSchema *customSchema;
```

Comment:
I mean, `RLMSchema` is already public, so I feel that would be more confusing? But it should be easy enough to change